src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -351,8 +375,16 @@ class OrganizationIntegrations extends AsyncComponent<
           onInstall={this.onInstall}
         />
         <SearchInput
-          value=""
-          onChange={() => {}}
+          value={this.state.searchInput || ''}
+          onChange={({target}) => {
```

Comment:
@NisanthanNanthakumar Let's pull this callback into a standalone function for readability